src/features/chains/utils.ts

```diff
@@ -1,19 +1,20 @@
+import { MultiProvider } from '@hyperlane-xyz/sdk';
+
 import { Environment } from '../../consts/environments';
-import { getMultiProvider } from '../../multiProvider';
 import { toTitleCase } from '../../utils/string';
 
-export function getChainName(chainId?: number) {
-  return getMultiProvider().tryGetChainName(chainId || 0) || undefined;
+export function getChainName(mp: MultiProvider, chainId?: number) {
```

Comment:
Nit: I guess multiProvider is a handful to write every time but it might be nice to be consistent